<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>attack_calculator</class>
 <widget class="QMainWindow" name="attack_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>561</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Attack Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/image_folder/smaller_icon.ico</normaloff>:/image_folder/smaller_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>150</width>
    <height>150</height>
   </size>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0" rowspan="2" colspan="3">
     <widget class="QListWidget" name="list_installed">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="btnReset">
      <property name="text">
       <string>Clear/Reset</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="lbl_CalcAttack">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Calculated Attack:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_CalcAttack">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btn_addSW">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add SW</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="list_rw">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinRW">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="list_sw">
        <property name="styleSheet">
         <string notr="true">background-image: url();</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinSW">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="6">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="lbl_TacOfficers">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Tactical Officers:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_TacOfficers">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn_addRW">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Add RW</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lbl_sw">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Special Weapons:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QPushButton" name="btn_Remove">
      <property name="text">
       <string>Remove Item</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="lbl_rw">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Researched Weapons:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Mutagenic Cartridge</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Stryll Pathogen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Kulgox Octal Sac</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="7" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="cbCerulean">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Cerulean Fighter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbNightclaw">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Nightclaw Fighter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbDarkSmuggler">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Dark Smuggler Chassis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="8">
     <widget class="QPushButton" name="btnCalculate">
      <property name="text">
       <string>Calculate</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lbl_installed">
      <property name="styleSheet">
       <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;Installed Modules:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_anubix">
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Anubix Drone:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboApparitious">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_apparitious">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Apparitious Ally:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_bioship">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">background-image: url(:/image_folder/transparent.png);</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Bioship Ally:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBioship">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboAnubix">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="4" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="cbACF">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Ancient Crystal Foci</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCrimsonObelisk">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Crimson Obelisk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCrimsonAmp">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Crimson Amplifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCruxAmp">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Crux Amplifier</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbShockPulse">
        <property name="text">
         <string>ShockPulse Charger</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>&amp;Save</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="btnLoad">
      <property name="text">
       <string>&amp;Load</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lbl_rw</zorder>
   <zorder>lbl_installed</zorder>
   <zorder>lbl_bioship</zorder>
   <zorder>lbl_apparitious</zorder>
   <zorder>lbl_anubix</zorder>
   <zorder>lbl_CalcAttack</zorder>
   <zorder>lbl_TacOfficers</zorder>
   <zorder>list_installed</zorder>
   <zorder>list_rw</zorder>
   <zorder>list_sw</zorder>
   <zorder>comboBioship</zorder>
   <zorder>comboApparitious</zorder>
   <zorder>comboAnubix</zorder>
   <zorder>ln_TacOfficers</zorder>
   <zorder>ln_CalcAttack</zorder>
   <zorder>cbACF</zorder>
   <zorder>cbCrimsonObelisk</zorder>
   <zorder>cbCrimsonAmp</zorder>
   <zorder>cbCruxAmp</zorder>
   <zorder>cbCerulean</zorder>
   <zorder>cbNightclaw</zorder>
   <zorder>cbDarkSmuggler</zorder>
   <zorder>spinRW</zorder>
   <zorder>spinSW</zorder>
   <zorder>cbACF</zorder>
   <zorder>cbCerulean</zorder>
   <zorder>btnCalculate</zorder>
   <zorder>btnReset</zorder>
   <zorder>checkBox_2</zorder>
   <zorder>checkBox_3</zorder>
   <zorder>checkBox</zorder>
   <zorder>btn_Remove</zorder>
   <zorder></zorder>
   <zorder>btn_addRW</zorder>
   <zorder>lbl_sw</zorder>
   <zorder>btn_addSW</zorder>
   <zorder></zorder>
   <zorder>btnSave</zorder>
   <zorder>btnLoad</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="action_Save"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionBase_Combat"/>
    <addaction name="actionInvasion"/>
    <addaction name="actionNPC"/>
    <addaction name="actionStandard"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionTutorial"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionStandard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Standard</string>
   </property>
  </action>
  <action name="actionNPC">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>NPC</string>
   </property>
  </action>
  <action name="actionInvasion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Invasion</string>
   </property>
  </action>
  <action name="actionBase_Combat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Base Combat</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
